### PR TITLE
Fix issue #1863, kve sort quirks, and bump ng-sortable

### LINKED ISSUE #1863: Can't pull second-to-last env var below last env var in key value editor
When reordering env vars, I can't pull the second-to-last value to the bottom. I have the pull the last value up.

@rhamilto FYI

### PR DESCRIPTION
**Pull request by benjaminapetersen**
- fix #1863 could not pull second-to-last env var below last env var in kve
- issue #620 possibly helped (?)
- bugzilla #1428991 reordering env vars down only works by twos
  - example: third moves to fifth, then seventh
- bugzilla #1369315 possibly helped (?)

I've mentioned 2 additional issues that may be helped by this fix as many quirky behaviors seem to resolve.  That said, I will test more before closing them.

@rhamilto @spadgett

### CODE REVIEW

File: dist/scripts/scripts.js [hunk 1]
```diff
@@ -13497,10 +13497,13 @@ uniqueForKey: r.uniqueForKey,
 uniqueForValue: r.uniqueForValue,
 dragControlListeners: {
 accept: function(e, t) {
-return e.itemScope.sortableScope.$id === t.$id;
+return console.log("accept?"), e.itemScope.sortableScope.$id === t.$id;
```
Comment (spadgett) [inline dist/scripts/scripts.js]: Looks like there is some debug logging in the dist

Comment (benjaminapetersen) [inline dist/scripts/scripts.js]: yup! got it.

**Comment by benjaminapetersen** [comment]
Rebased.  This one is def improvements on kve behavior.

**Comment by spadgett** [comment]
[merge]

**Comment by spadgett** [comment]
@benjaminapetersen Can you test which are the bugs are really fixed so we can update the Bugzilla status?

**Comment by benjaminapetersen** [comment]
@spadgett yup.  I think the ones with the (?) are not sufficient, but the other ones are good.  I'll give them another test run to make sure.

**Pull request merged**